DEPENDENCIES.yaml

```diff
@@ -0,0 +1,4 @@
+Common Name:
+  - EmoDB/Rison
```

Comment:
We don't really own this package.  I don't mind the dependencies, but maybe common name should just be "Rison" or "Java/Rison" or something, just not with "EmoDB" in it.

Reply:
I can change it to Bazaarvoice/Rison since it's our fork.
The OWENER file has us listed..just FYI

Reply:
Ooooh... interesting.  I still like "Bazaarvoice/Rison" better.  Even if Emo is the primary (perhaps only) user there's nothing in this package that's Emo-specific.